public/resources/css/module/_resources.scss

```diff
@@ -1,10 +1,10 @@
 .resources {
   display: flex;
   justify-content: space-between;
-  flex-wrap: wrap;
   flex-direction: row-reverse;
 
-  @media(max-width: 400px) {
+  @media(max-width: 768px) {
```

Comment:
It gets a bit claustrophobic when the window is resized, so I increased the breakpoint to turn into a column sooner